apps/total-typescript/src/templates/tutorial-template.tsx

```diff
@@ -147,10 +183,17 @@ const Header: React.FC<{tutorial: SanityDocument}> = ({tutorial}) => {
   )
 }
 
-const TutorialExerciseNavigator: React.FC<{tutorial: SanityDocument}> = ({
+const TutorialExerciseNavigator: React.FC<{tutorial: Module}> = ({
   tutorial,
 }) => {
-  const {slug, lessons, _type} = tutorial
+  const {slug, lessons: _lessons, sections, _type} = tutorial
+  const {data: moduleProgress} = trpc.moduleProgress.bySlug.useQuery({
+    slug: tutorial.slug.current,
+  })
+
+  const lessons =
+    sections && sections.length === 1 ? sections[0].lessons : _lessons
```

Comment:
did some preparation here for when we force sections on content level so that it'll keep working just fine.